v2/de/de_holidays.go

```diff
@@ -44,6 +44,16 @@ var (
 	// Fronleichnam represents Corpus Christi on the 60th day after Easter
 	Fronleichnam = aa.CorpusChristi.Clone(&cal.Holiday{Name: "Fronleichnam", Type: cal.ObservancePublic})
 
+	// Friedensfest represents the Augsburger Hohes Friedensfest on 8-Aug
+	Friedensfest = &cal.Holiday{
+		Name:      "Friedensfest",
+		Type:      cal.ObservancePublic,
+		Month:     time.August,
+		Day:       8,
+		Func:      cal.CalcDayOfMonth,
+		StartYear: 1650,
```

Comment:
Should be 1950?
> Seit 1950 ist das Augsburger „Hohe Friedensfest“ im Stadtkreis Augsburg gesetzlicher Feiertag

Reply:
Maybe add a `HolidaysAugsburg []*cal.Holiday` collection

Reply:
Depends on the definition of StartYear. Is StartYear the first time the holiday was observed or the first time that it is a legal holiday?

Reply:
It's used to determinate if a date is a holiday. 8.8.1949 wasn't a holiday and based on the code comment ("the first year the holiday is observed") it would make sense that it's the second. Compare it with the "Weltkindertag" it's a "day" since the fifties but a holiday since 2019.